### PR TITLE
Dev -> Master for 2.11.1 

### PR DESCRIPTION
**Pull request by mashehu**

### CODE REVIEW

File: docs/images/nf-core-modules-patch.svg
Comment (maxulysse) [inline docs/images/nf-core-modules-patch.svg:1]: this one had changed a lot, is that expected?

File: docs/images/nf-core-modules-test.svg
Comment (maxulysse) [inline docs/images/nf-core-modules-test.svg:1]: this one had changed a lot, is that expected?

File: docs/images/nf-core-subworkflows-info.svg
Comment (maxulysse) [inline docs/images/nf-core-subworkflows-info.svg:1]: this one had changed a lot, is that expected?

File: docs/images/nf-core-subworkflows-test.svg
Comment (maxulysse) [inline docs/images/nf-core-subworkflows-test.svg:1]: this one had changed a lot, is that expected?

**Comment by maxulysse** [comment]
how come we have the message:
> `This branch is out-of-date with the base branch`

File: .gitpod.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-image: nfcore/gitpod:latest
+image: nfcore/gitpod:dev
```
Comment (mirpedrol) [inline .gitpod.yml:1]:
```suggestion
image: nfcore/gitpod:latest
```

File: docs/images/nf-core-modules-patch.svg
Comment (mashehu) [reply docs/images/nf-core-modules-patch.svg:1]: yep, all of them are improvements to before 🙂

**Pull request merged**